### DISCUSSION TITLE
[Itau] Connect to URL filter failed em ambiente de sandbox

### DISCUSSION THREAD

**Discussion by mattshuegel** [bacen/pix-api#400] [General]
Pessoal, boa tarde!
Estou tentando fazer alguns testes relacionados ao ITAU, porém o mesmo está sempre retornando o erro "Connect to URL filter failed" ao tentar registrar uma transação de PIX (https://api.itau.com.br/sandbox/pix_recebimentos/v1/cob/7117fcf9915449d8a36bd0f3fad223ce)

Segue abaixo o PUT que estou fazendo:
curl --location --request PUT 'https://api.itau.com.br/sandbox/pix_recebimentos/v1/cob/7117fcf9915449d8a36bd0f3fad223ce' \
--header 'Content-Type: application/json' \
--header 'x-itau-flowID: 1' \
--header 'x-correlationID: 2' \
--header 'Authorization: Bearer TOKEN' \
--data-raw '{
    "chave": "sandbox@itau-unibanco.com.br",
    "calendario": {
        "expiracao": "86400"
    },
    "devedor": {
        "cpf": "17183579077",
        "nome": "JOAO DA SILVA"
    },
    "valor": {
        "original": "10,00"
    }
}'

Alguém já passou por isso?

**Comment by rubenskuhl** [comment]
O endereço é mesmo api.itau.com.br ? O serviço de produção deles fica em api.itau .

**Comment by mattshuegel** [comment]
> O endereço é mesmo api.itau.com.br ? O serviço de produção deles fica em api.itau .

Esse é o endereço de sandbox. O de produção é outro.

**Comment by rubenskuhl** [comment]
> > O endereço é mesmo api.itau.com.br ? O serviço de produção deles fica em api.itau .
> 
> Esse é o endereço de sandbox. O de produção é outro.

Mas acho que vale tentar com api.itau/sandbox, api.gtm.itau.com.br/sandbox e api.gtm.itau.com.br sem sandbox e ver o que dá.

**Comment by duduccosta** [comment]
Qual a URL de produção?
